cfme/utils/appliance/__init__.py

```diff
@@ -2451,6 +2451,8 @@ def disable_freeipa(self, log_callback=None):
         """Switch UI back to database authentication, and run --uninstall-ipa on appliance"""
         self.appliance_console_cli.uninstall_ipa_client()
         self.server.authentication.configure(auth_mode='database')
+        # reset ntp servers
+        self.server.settings.update_ntp_servers({'ntp_server_1': 'clock.corp.redhat.com'})
```

Comment:
Should we fetch value for `ntp_server_1` from yamls?

Reply:
`update_ntp_servers` is a wrapper for a fill method, so `ntp_server_1` the name of the widget we're filling.